src/core_plugins/elasticsearch/lib/determine_supported_features.js

```diff
@@ -0,0 +1,25 @@
+import { checkFieldStatsApi } from './feature_checks';
+
+
+const featureChecks = [
+  checkFieldStatsApi,
+];
+
+export async function determineSupportedFeatures(callAdminAsKibanaUser) {
+  const { nodes: nodesInfo } = await callAdminAsKibanaUser('nodes.info', {
```

Comment:
But it is part of the health check. So this looks like the same request mechanism that the rest of the code uses.